### ISSUE TITLE
Types FROM and TO may be misleading

### ISSUE THREAD

**Issue by MaciejCiemiega** [mobnetic/compose-shared-element#2]
The types names may be a bit misleading, because they can be used interchangeably.
It doesn't matter if first one if `FROM` or `TO`. They are only used to detect the "other" one from the pair.

Without it I have no idea how to detect that one element was on "previous" screen and second one is on "next" screen, because there is no concept of screens and transitions between them.
We cannot rely on things like position or hierarchy change. Not every hierarchy change is a "screen" change and that would make it transition from its old position to its new position (or old/new places in hierarchy).

 Maybe a better idea would be to change it to something more generic like `screen: String` so one can specify values like:
- "list" on one screen
- "details" on second screen

For now I stayed with `FROM` and `TO` because it makes it super simple to choose one o two enum values.

**Comment by myxzlpltk** [comment]
Yeah its true. Maybe if it use tag property just like hero widget it wil be better
